CodeReview/Maru_code_forReview.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+from netCDF4 import Dataset
+
+
+def ferry_data_download(URL):
```

Comment:
There two ferries with different URLs, for the project we have been using one, but for the future we might use two, so I will leave the URL as an input. Also, they might change the location of the files.